project3/styles.css

```diff
@@ -0,0 +1,140 @@
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+
+}
+
+
+
+body {
+    background-color: var(--color-blue);
+    color: var(--color-offwhite);
+    font-family: 'BenchNine', sans-serif;
+
+
+}
```

Comment:
This is a great start to your stylesheet and these three things are 💯 in the order I'd expect them to be in -- great job!